assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -290,36 +291,36 @@ function MyStories() {
   return (
     <FontProvider>
       <TransformProvider>
-        <UnitsProvider pageSize={pageSize}>
-          <Layout.Provider>
-            <Layout.Squishable>
-              <PageHeading
-                defaultTitle={__('My Stories', 'web-stories')}
-                searchPlaceholder={__('Search Stories', 'web-stories')}
-                filteredStories={orderedStories}
-                handleTypeaheadChange={handleTypeaheadChange}
-                typeaheadValue={typeaheadValue}
-              >
-                <ToggleButtonGroup
-                  buttons={STORY_STATUSES.map((storyStatus) => {
-                    return {
-                      handleClick: () =>
-                        handleFilterStatusUpdate(storyStatus.value),
-                      key: storyStatus.value,
-                      isActive: status === storyStatus.value,
-                      text: storyStatus.label,
-                    };
-                  })}
-                />
-              </PageHeading>
-              {storiesViewControls}
-            </Layout.Squishable>
-            <Layout.Scrollable>{BodyContent}</Layout.Scrollable>
-            <Layout.Fixed>
-              <ScrollToTop />
-            </Layout.Fixed>
-          </Layout.Provider>
-        </UnitsProvider>
+        <Layout.Provider>
+          <Layout.Squishable>
+            <PageHeading
+              defaultTitle={__('My Stories', 'web-stories')}
+              searchPlaceholder={__('Search Stories', 'web-stories')}
+              filteredStories={orderedStories}
+              handleTypeaheadChange={handleTypeaheadChange}
+              typeaheadValue={typeaheadValue}
+            >
+              <ToggleButtonGroup
+                buttons={STORY_STATUSES.map((storyStatus) => {
+                  return {
+                    handleClick: () =>
+                      handleFilterStatusUpdate(storyStatus.value),
+                    key: storyStatus.value,
+                    isActive: status === storyStatus.value,
+                    text: storyStatus.label,
+                  };
+                })}
+              />
+            </PageHeading>
+            {storiesViewControls}
+          </Layout.Squishable>
+          <Layout.Scrollable>
+            <UnitsProvider pageSize={pageSize}>{BodyContent}</UnitsProvider>
```

Comment:
Only change here is I moved the `UnitsProvider` for grid/table here instead of wrapping the whole page, following suit from detail template view related templates grid.